Resources/Private/Modules/Views/Index.fusion

```diff
@@ -3,7 +3,7 @@ Garagist.Mautic.BackendController.index = Neos.Fusion:Component {
     hasCategories = ${Carbon.Array.check(categories)}
     showPagesWithoutEmails = ${this.hasPages && Array.some(pages, page => page.count > 0) ? 'false' : 'true'}
     logo = ${ping ? 'Logo.svg' : 'LogoInactive.svg'}
-    mauticPublicUrl = ${Configuration.Setting('Garagist.Mautic.publicUrl')}
+    mauticPublicUrl = ${Mautic.getPublicUrl(node)}
```

Comment:
I don't think we have to value node here:

https://github.com/Garagist/Garagist.Mautic/blob/main/Classes/Controller/BackendController.php#L224-L230